### ISSUE TITLE
Notistack Snackbar toast messages fail accessibility

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1269]
TLDR: notistack sucks because its too fast

The Notistack Snackbar alerts fails multiple success criterion:

1.4.3 (AA) Insufficient color contrast https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html

2.2.1 (AA) Users can complete tasks without unexpected changes in content or context that are a result of a time limit. https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html

2.2.3 (AAA) Timing is not an [essential](https://www.w3.org/WAI/WCAG21/Understanding/no-timing.html#dfn-essential) part of the event or activity presented by the content, except for non-interactive [synchronized media](https://www.w3.org/WAI/WCAG21/Understanding/no-timing.html#dfn-synchronized-media) and [real-time events](https://www.w3.org/WAI/WCAG21/Understanding/no-timing.html#dfn-real-time-event). 

4.1.3 (AA) Users are aware of important changes in content that are not given focus, and to do so in a way that doesn't unnecessarily interrupt their work. https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html
